verilog/tools/ls/autoexpand_test_format

```diff
@@ -0,0 +1,47 @@
+#!/bin/env python3
+# Formats all raw strings in autoexpand_test.cc using Verible's formatter.
```

Comment:
This looks like a script that was helpful in the first transition, but now using the formatter directly from within the test should be sufficient. So I'd not include this script here.

Reply:
I think it could be useful if a developer adds/edits a test, or the default formatting rules change. The formatter check gives you a diff, but you still have to format the code manually.

Reply:
That makes sense. I'll get rid of the script.